Makefile

```
@@ -33,13 +33,9 @@ docker-image: ## Build docker image.
 
 ##@ Test
 
-.PHONY: unit-test
-unit-test: ## Run unit test.
-	cargo test --workspace
-
-.PHONY: integration-test
-integration-test: ## Run integation test.
-	cargo test integration
+.PHONY: nextest
```

suggestion: I think we need to add the `nextest` target to install the toolchain and use the `tests` target:

```suggestion
tests: nextest ## Run unit and integration tests.
	cargo nextest run

.PHONY: nextest ## Install nextest tools.
nextest:
	cargo --list | grep nextest || cargo install cargo-nextest --locked
```